scripts/setup-remote-cluster.sh

```
@@ -130,10 +124,16 @@ retry_until 600 15 "oc ${remote} get hyperconverged kubevirt-hyperconverged -n o
     -o jsonpath='{.status.conditions[?(@.type==\"Available\")].status}' 2>/dev/null | grep -q True" || { echo "Timed out waiting for CNV to be available"; exit 1; }
 echo "CNV ready"
 
+else
+echo "Skipping prerequisites (LVMS, CNV) -- SKIP_PREREQUISITES=true"
+fi
+
 # Remote cluster: prepare for OSAC
 
+REMOTE_STORAGE_CLASS=${REMOTE_STORAGE_CLASS:-"lvms-vg1"}
+
 oc ${remote} create namespace ${INSTALLER_NAMESPACE}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)